### PR TITLE
Add pokemon weaknesses to raid status messages

### PR DESCRIPTION
**Pull request by Googer**

### CODE REVIEW

File: app/pokemon.js [hunk 1]
```diff
@@ -74,6 +77,42 @@ class Pokemon extends Search {
...
+
+		return Object.keys(types)
+			.map(type => {
+				let multiplier = 1.0;
+
+				pokemon_types.forEach(pokemon_type => {
+					if (types[type].se.includes(pokemon_type)) {
+						multiplier *= 1.400;
+					} else if (types[type].ne.includes(pokemon_type)) {
+						multiplier *= 0.714;
```
Comment (ForcedUnknown) [inline app/pokemon.js:94]: Not that it really matters, but where did these multiply values come from?

**Comment by ForcedUnknown** [comment]
@Googer how do you know what type each pokemon is?  I don't see any changes to the pokemon.json file? I feel like I must be missing something.

File: app/pokemon.js (hunk 1 above)
Comment (Googer) [reply app/pokemon.js:94]: They're the current-accepted values that Pokemon Go uses - I believe they come straight from its game_master file.

**Comment by Googer** [comment]
The pokemon.json file is a huge diff to add the types to everything - github just isn't showing it unless you ask it to since it hides large diffs by default.

File: app/pokemon.js (hunk 1 above)
Comment (ForcedUnknown) [reply app/pokemon.js:94]: Ah nice, I think Ranked Boosted (where I look for stuff, or rather used to look) is just suuuuuuper behind the times.  They gave up shortly before that lapras nerf haha.

**Comment by ForcedUnknown** [comment]
Ah bingo, you're right, was hiding it really well haha

**Pull request merged**